checkov/common/output/github_actions_record.py

```diff
@@ -23,6 +23,7 @@ def __init__(self,
                  entity_tags: Optional[Dict[str, str]] = None,
                  bc_check_id: Optional[str] = None,
                  severity: Optional[Severity] = None,
+                 details: List[str] = []
```

Comment:
```suggestion
                 details: Optional[List[str]] = None
```
never use a mutable object, such as `{}` or `[]` as a default value for method parameters, otherwise strange things can happen, because the object is actually only once created, even you call it several times.

Reply:
makes sense, guess the method signature is evaluated once and cached?